notes_commands.py

```diff
@@ -2,8 +2,17 @@
 from notes_decorator import input_error
 
 
+def print_note(key, value):
+    result = 20 * "-" + "\n"
+    result += f"note id - {key}\n"
+    result += f"note text - {value.note_text}\n"
+    if value.note_tags:
+        result += f"tags - {' '.join(tag for tag in value.note_tags)}\n"
```

Comment:
Прийме цю пропозицію. Я подумав, що буде правильніше и читабельніше, якщо виводити теги в нотатці в алфавітному порядку. Так для користувача буде зрозуміліше.

```suggestion
        result += f"tags - {' '.join(sorted(tag for tag in value.note_tags))}\n"
```